From 17a2b1e5ed172a5a009f8e5e01fb1a3d1bb4e72a Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Sat, 19 Aug 2017 12:30:07 +0800
Subject: [PATCH 04/36] ARM: sun8i: q8-1024x600: Enable LCD with a generic
 panel

Enable the LCD on 1024x600 Q8 tablets with a generic panel compatible.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm/boot/dts/sun8i-a33-q8-tablet-1024x600.dts | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm/boot/dts/sun8i-a33-q8-tablet-1024x600.dts b/arch/arm/boot/dts/sun8i-a33-q8-tablet-1024x600.dts
index 6f8ebd6c90ee..76f99611e182 100644
--- a/arch/arm/boot/dts/sun8i-a33-q8-tablet-1024x600.dts
+++ b/arch/arm/boot/dts/sun8i-a33-q8-tablet-1024x600.dts
@@ -47,3 +47,16 @@ / {
 	compatible = "allwinner,q8-a33-1024x600", "allwinner,q8-a33",
 		     "allwinner,sun8i-a33";
 };
+
+&de {
+	status = "okay";
+};
+
+&panel {
+	compatible = "innolux,zj070na-01p";
+	status = "okay";
+};
+
+&tcon0 {
+	status = "okay";
+};
-- 
2.30.2

